docs/conf.py

```diff
@@ -278,25 +278,16 @@
 latex_elements = {
     'papersize': 'a4',
     'pointsize': '11pt',
-    'figure_align': 'tbp',
+    'figure_align': 'H',
```

Comment:
My body stopped twitching now....

In my experience, the desire to force the placement of figures in a PDF is almost always in indication of suboptimal figure captions. A document often suffers substantially in readability when figures are not placed into the normal locations: at the top, the bottom, or covering a full page.

Reply:
I made the change after trying unsuccessfully to make it work in another way. This for example is what the previous configuration did to the "what you really need to know" part (left: old, right: with the change in this PR) - it just puts 4 of the images right after another, out of the context of the small text box it applies to:

![image](https://user-images.githubusercontent.com/29738718/72742509-2b96be80-3baa-11ea-84d0-ded00803935e.png)

Here is another one (again, left = old, right = with change in this PR): there are two screenshots, on the right you can see that the text starting with "The above screenshot" should be below the screenshot, and is helpful to visually distinguish the two screenshots (on the right), but it currently is above the screenshots.

![image](https://user-images.githubusercontent.com/29738718/72742783-baa3d680-3baa-11ea-91f0-4f3333a8db28.png)

I could indeed make use of subtitles for the figures to fix this, but I don't think this would guarantee correct figure placement in all cases.

Reply:
I actually scrolled through the whole PDF after your comment, and only then I really noticed how many figures are funnily messed up with the current configuration. In any suboptimal case, the "H" configuration improves the visual experience and the comprehensiveness of the contents, and there is no figure where it makes things worse.
Obviously, there are still plenty of problems with the figures. Some figures resolution is really low, and the sizing of many figures seems to be off. 

If you have ideas on how to improve on the current state of figure placement I'm happy to hear them, but I think this small change really goes a long way for very little effort.

Reply:
OK :raised_hands: ;-)

Let's assume I never said anything ;-)

(we will never be perfect with an autogenerated PDF, and I will now go and sacrifice three oxen to Donald Knuth ... that should get me out of the guilt mode)

Reply:
I'm so sorry for the pain this inflicts, and my blasphemy ;-) you can always blame me for this ;-)

Reply:
;-) I will survive. The PDF is objectively better than before with this change applied.